lib/args.test.ts

```diff
@@ -112,6 +120,20 @@ describe("format_args()", function() {
     expect(readFixed(buffer, 0)).to.equal(3.5);
   });
 
+  it("encodes empty array argument", function () {
+    const args = [new Uint8Array([])];
+    const def: ArgumentDefinition[] = [{ name: "arg1", type: "array" }];
+    const buffer = format_args(args, def);
+    expect(buffer.toString("hex")).to.equal("00000000");
+  });
```

Comment:
Found these empty tests helpful when debugging. This does not branch on zero-size, but still useful to ensure we dont add extra bytes.